src/components/Modal/ConfirmDeleteModal.tsx

```diff
@@ -0,0 +1,66 @@
+import { commentDeleteState } from '@/store/recoilCommentState';
+import {
+  confirmDeleteCommentModalState,
+  confirmDeleteReviewModalState,
+} from '@/store/recoilModalState';
+import { reviewDeleteState } from '@/store/recoilReviewState';
+import { useRecoilState, useSetRecoilState } from 'recoil';
+import ModalPortal from './ModalPortal';
+
+type ConfirmDeleteModalType = {
+  target: string;
+  text: string;
+};
+
+const ConfirmDeleteModal = ({ target, text }: ConfirmDeleteModalType) => {
+  const [commentDeleteOpen, setCommentDeleteOpen] = useRecoilState<boolean>(
```

Comment:
```suggestion
  const [commentDeleteOpen, setCommentDeleteOpen] = useRecoilState(
```
p4;

Recoil 정의하는 파일에 타입을 지정해서 현재 파일에서는 타입 추론을 추천합니다.